otter/models/intents.py

```diff
@@ -0,0 +1,16 @@
+from characteristic import attributes
+
+from effect.twisted import deferred_performer
+
+
+@attributes(['scaling_group', 'modifier'])
+class ModifyGroupState(object):
```

Comment:
Have you considered a generic `GroupModelRequest` intent as described [here](https://github.com/rackerlabs/otter/issues/844#issuecomment-70881324)? I feel it can be good path to move any code in direction of effect.